README.md

```diff
@@ -1 +1,65 @@
-# pediatric_cancer_atlas_profiling
\ No newline at end of file
+# Pediatric cancer atlas image profiling
+
+In this repository, we perform image analysis to extract single cell morphological profiles and image-based profiling to format the profiles to be processed with machine learning and other analyses.
+
+## Cell Painting
+
+We performed a traditional [Cell Painting assay](https://www.nature.com/articles/nprot.2016.105) on multiple different pediatric cancer cell lines.
+Images were acquired using an Opera Phenix 1 High Content Screening System.
+
+In this assay, we have six channels (five Cell Painting + Brightfield). 
+These channels are listed in the number order that you can find for the TIFF images, with the channel name in the XML file bolded:
+
+1. **Brightfield high**
+2. Concanavalin A **Alexa 488** - Endoplasmic reticulum (ER)
+3. Phalloidin and WGA **Alexa 568** - Actin, Golgi, and plasma membrane (AGP)
+4. MitoTracker **Alexa 647** - Mitochondria
+5. **Hoechst 33342** - DNA/nucleus
+6. SYTO14 **Alexa 488 long** - Cytoplasmic RNA and nucleoli 
+
+![Example Cell Painting](./examples/cell_painting_alsf.png)
+
+## Data
+
+### Pilot dataset
+
+To assess the optimal seeding density and time point across all cell lines, we acquired six plates of preliminary data with 19 unique cell lines.
+There are two platemap layouts, with each layout having three time points (24, 48, and 72 hours) per plate (e.g., 2 layouts * 3 time points = 6 plates).
+Per plate, there are five different seeding densities (1000, 2000, 4000, 8000, 12000).
```

Comment:
Consider specifying if these seeding densities are per FOV or per well. I assume well.

Reply:
Yup it is at the well level! The plate maps below demonstrate this but I can also add that in the documentation.